index.html

```diff
@@ -223,6 +255,22 @@ <h2>
             </li>
           </ol>
         </li>
+        <li>If |input|'s {{ImageResource/label}} member is present and not the
+        empty string:
+          <ol>
+            <li>Set |image|'s [=image resource/label=] to
+            |input|.{{ImageResource/label}}.
+            </li>
+          </ol>
+        </li>
+        <li>If |input|'s {{ImageResource/label}} member is not present and an
```

Comment:
Wait, doesn't substitution assume the resource is fetched? If so, that probably shouldn't be part of this algorithm (same below).

Reply:
That makes sense, substitution only needs to happen if that image resource is selected for use. Where would you capture that? It's not part of the parsing algorithm, so it's party of the… ?

Reply:
This goes back to this being somewhat wishful thinking/false hope... it's not something implementations will do (even with SVGs, they will be treated as static images). I don't think we should mention it at all to be honest. The right thing to do is to add the labels or let other parts (e.g., short_name) do the right thing.